examples/01_forecasting.ipynb

```diff
@@ -2513,7 +2513,13 @@
    "source": [
     "###  3.4.3 simple ensembling strategies\n",
     "\n",
-    "TODO - contributions in this section are appreciated"
+    "`sktime` provides `EnsembleForecaster` which can ensemble point forecasts generated by multiples forecasters using `aggfunc`. \n",
```

Comment:
Thanks @AngelPone - I appreciate the detail here, my only worry is that it's hard to keep this up-to-date. Perhaps a better place for these details would be the docstring of the class? Adding the reference to the `EnsembleForecaster` in this list is a good idea!

Reply:
yes, but that could be a separate contribution.

It´s really nice to have this in the tutorial, especially since we asked for help.
Thanks a lot, @angelpone!

Reply:
@mloning  I also added docstring to `EmsembleForecaster`, but I don't know how to add the reference to `EmsembleForecaster` in notebook.

Reply:
oops, I just wanted to look at this and merge it, but your newer contribution breaks the doc tests - would you mind having a look at it, @angelpone?